app/javascript/dashboard/store/modules/contactConversations.js

```diff
@@ -76,6 +76,26 @@ export const getters = {
   getContactConversation: $state => id => {
     return $state.records[Number(id)] || [];
   },
+  getPreviousConversation: $state => (contactId, conversationId) => {
+    const conversations = $state.records[Number(contactId)] || [];
+    const sortedConversations = conversations.sort(
+      (a, b) => new Date(b.created_at) - new Date(a.created_at)
+    );
```

Comment:
```suggestion
    const sortedConversations = conversations.sort(
      (a, b) => new Date(b.created_at) - new Date(a.created_at)
    );
```

Do we need to care about the order of conversations in other components?  `sort` is an in-place sort function, it can modify the original array